app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -101,14 +127,15 @@ Template.visitorForward.events({
 	},
 
 	'change #forwardDepartment, blur #forwardDepartment'(event, instance) {
-		if (event.currentTarget.value) {
-			instance.find('#forwardUser').value = '';
+		const [user] = instance.selectedAgents.get();
+		if (user) {
+			instance.selectedAgents.set([]);
 		}
 	},
 
 	'change #forwardUser, blur #forwardUser'(event, instance) {
 		if (event.currentTarget.value && instance.find('#forwardDepartment')) {
-			instance.find('#forwardDepartment').value = '';
+			instance.selectedDepartments.set([]);
```

Comment:
This is not working

Reply:
Done. It was implemented to listen to the old select component.